scripts/generate_ssl_cert

```diff
@@ -64,14 +64,11 @@ openssl req -batch -new -key privkey.rsa -days 3650 -config config -out cert.csr
 # the extensions too, but "req" does not allow the -extfile option.)
 openssl x509 -extfile config -req -signkey privkey.rsa -in cert.csr -outform PEM -out signedpubcert.pem -days 3650
 
-openssl dhparam 512 > dh.pem
-
 popd
 
 tmpcert="${DIR}/tmpcertcombined.pem"
```

Comment:
Should this use `mktemp`?

Reply:
Better to use it, even if It's bound to be replaced by the ocaml code

Reply:
privkey.rsa and signedpubcert.pem should also use mktemp as well.
The fact that they do not get deleted after they get consolidated into the single cert is also iffy.

I wonder if something else depends on those files being present.